packages/daimo-api/src/network/bundlerClient.ts

```diff
@@ -243,6 +175,23 @@ function packGasFees({
 function packHiLo(hi: bigint, lo: bigint) {
   return concatHex([
     numberToHex(hi, { size: 16 }),
-    numberToHex(lo, { size: 32 }),
+    numberToHex(lo, { size: 16 }),
```

Comment:
bundle compression no longer immediately necessary for two reasons:
- calldata is cheap for now, blob data priced @ nearly free
- ERC-4337 v0.7 has a less-egregious userop serialization format